src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -1480,6 +1488,32 @@ public boolean isPaidLessThanClaimAmount() {
     @JsonIgnore
     public boolean isCourtDecisionInClaimantFavourImmediateRePayment() {
         return hasApplicant1CourtDecisionInFavourOfClaimant()
-                && getApplicant1RepaymentOptionForDefendantSpec() == PaymentType.IMMEDIATELY;
+            && getApplicant1RepaymentOptionForDefendantSpec() == PaymentType.IMMEDIATELY;
+    }
+
+    /**
+     * LiP applicant requires Welsh if they selected Welsh or Both languages.
+     *
+     * @return true if applicant is LiP and they accept Welsh (or English+Welsh)
+     */
+    @JsonIgnore
+    public boolean isLipApplicantRequiringWelsh() {
+        return Optional.ofNullable(getClaimantBilingualLanguagePreference())
+            .filter(value -> List.of(Language.WELSH.toString(), Language.BOTH.toString()).contains(value))
+            .isPresent();
+    }
+
+    /**
+     * LiP applicant requires Welsh if they selected Welsh or Both languages.
```

Comment:
See comment above. If it does turn out we need this method, be sure that the comment reflects the use of respondent rather than Applicant.